source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -254,15 +254,20 @@ def updated_common_journey_request_with_default(request, instance):
         request['car_speed'] = instance.car_speed
 
 
+def walking_in_origin_destination_mode(request):
+    return 'walking' in request['origin_mode'] and 'walking' in request['destination_mode']
+
+
 def updated_request_with_default(request, instance):
     updated_common_journey_request_with_default(request, instance)
 
-    if request['_min_car'] is None:
+    if request['_min_car'] is None and walking_in_origin_destination_mode(request):
```

Comment:
So we apply the min only if there is walking in both origin AND destination?
@TeXitoi I think your opinion is welcome on that.

Reply:
yes, including the default values